### DISCUSSION TITLE
Run renovatebot only to rebase pr's

### DISCUSSION THREAD

**Discussion by florisluiten** [renovatebot/renovate#10570] [Request Help] [answered]
I have renovatebot running in a self-hosted gitlab environment, where it runs every hour. That's fine for creating new pr's.

However, from time-to-time I want a pr to be rebased by renovatebot. So I check the "If you want to rebase/retry this MR, check this box." box, and then have to wait a maximum of 60 minutes before renovatebot 'comes along' to do the rebasing. I could run renovatebot more often, but do not want it to the the "full run" more than once per hour (since it takes around 7 minutes to complete).

Is it possible to run renovatebot, so that it only rebases pr's (when needed), without checking/creating new pr's?

**Comment by rarkins** [comment, marked as answer]
It's not currently supported. If you were to run https://github.com/whitesource/renovate-on-prem instead though, it can respond to webhooks
